### ISSUE TITLE
[Forge Mod] DBC Breaks the camera

### ISSUE THREAD

**Issue by zeshinumi** [jrbudda/minecrift#68]
I'm working with the version of 17 that included the fix for pixelmon camera breaking the game when the battle ends. I loaded up the game with the Dragon block C mod and the camera goes completely wonky, and all I can see is blue sky. I jumped back to version 15 and the game just crashes upon loading the world. Pastebin of crash log here: http://pastebin.com/LxcdEKaT
My guess it has something to do with what that mod is trying to access to do their hair rendering

**Comment by jrbudda** [comment]
still doing it in v21?

**Comment by zeshinumi** [comment]
I'll test again when I get the chance, I think last time I tried it crashed with v22

**Comment by zeshinumi** [comment]
Tested it against v21, crashes on loading a world
http://pastebin.com/x13DcfZU
Tested it against v24, same thing, crashes on world load
http://pastebin.com/JKkenedp

**Comment by Techjar** [comment]
Oh, he's hacking into updateCameraAndRender... and totally screwing with the camera transform and calling renderWorld in a different place by the looks of it. There's nothing we can do about that, because that method is how we actually do all the rendering, not to mention the functional incompatibility here since he's messing with the camera.

**Comment by jrbudda** [comment]
same actual error as LOTR, he must also be replacing the entityRenderer

zeshinumi could you try it with the build linked at the bottom of #41 ? the installation instructions for that mod make my brain hurt.

**Comment by zeshinumi** [comment]
Will do, if I were to suggest to them a better way to render things like hair models and other player entity changes, would their be a better way to make it compatible?

**Comment by jrbudda** [comment]
thats not likely the problem. hair should not affect vivecraft.

**Comment by zeshinumi** [comment]
tried the build from #41 and it runs, however it doesn't render the hairs or custom skins. I take it this is something the modder would have to change to make it work?

**Comment by jrbudda** [comment]
not necessarily, it's totally a case-by-case thing. we'd have to look and see what they're trying to do and if we're calling the Forge code correctly.
